packages/core/src/Stores/client-store.js

```diff
@@ -433,12 +434,20 @@ export default class ClientStore extends BaseStore {
 
         reaction(
             () => [this.account_settings],
-            () => {
-                const language = getRedirectionLanguage(this.account_settings?.preferred_language);
+            async () => {
+                const language = getRedirectionLanguage(this.account_settings?.preferred_language, this.is_new_session);
```

Comment:
what does "is_new_session" mean? is it signup flow?

Reply:
Not it means whether user is logged in or not